Cargo.toml

```diff
@@ -28,5 +27,8 @@ num_cpus = "1.8"
 rand = "0.8"
 rand_xorshift = "0.3"
 
+[lib]
+doc-scrape-examples = false
```

Comment:
What does this do?

Reply:
This it to opt-out of the `Scraping examples`, you can find more info here: <https://doc.rust-lang.org/rustdoc/scraped-examples.html>